### PR TITLE
Adding flag: drop tcp connections

### PR DESCRIPTION
**Pull request by Karen-Schoener**

### CODE REVIEW

**Review by gabordozsa — CHANGES_REQUESTED** [review]
@Karen-Schoener I cannot see how the value of the new flag gets into the intialRouterConfig. I would expect some extra code around here: https://github.com/skupperproject/skupper/blob/fca6f869c89dc83ccadc74202a9a137e1a732ea9/pkg/qdr/qdr.go#L72.  I think the new flag  should be handled similarly to `DataConnectionCount` parameter and it needs to be added to the `routerConfig.Metadata` here.

File: api/types/client.go [hunk 1]
```diff
@@ -67,15 +67,16 @@ type Tuning struct {
...
+	Logging                  []RouterLogConfig
+	MaxFrameSize             int
+	MaxSessionFrames         int
+	DataConnectionCount      string
+	IngressHost              string
+	ServiceAnnotations       map[string]string
+	PodAnnotations           map[string]string
+	LoadBalancerIp           string
+	DisableMutualTLS         bool
+	CloseConnectionsOnDelete bool
```
[the remark this answers is not in the document]
Comment (ajssmith) [reply api/types/client.go:79]: In perusing similar settings in firewall implementations, we might consider a flag name along the lines of:

DropActiveTcpConnections
or
TerminateActiveTcpConnections

I would lean towards not having the "OnDelete" suffix as the action is implicit and will be reflected in documentation, etc.

[the remark this answers is not in the document]
Comment (ajssmith) [reply api/types/client.go:79]: Or more simply "DropTcpConnections" as the active is implied...

**Comment by ajssmith** [comment]
Agree with @gabordozsa that update to the initial router config should be part of this change. As identified, should be addition to setting of RouterConfig metadata in qdr.go

**Comment by Karen-Schoener** [comment]
> Agree with @gabordozsa that update to the initial router config should be part of this change. As identified, should be addition to setting of RouterConfig metadata in qdr.go

Added new configurable to RouterMetadata. 

I will plan to make another checkin with the suggestion: rename param to DropTcpConnections.

File: api/types/client.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (Karen-Schoener) [reply api/types/client.go]: Renamed new param to: DropTcpConnections

**Review by gabordozsa — APPROVED** [review]
LGTM

**Pull request merged**